packages/components/src/utils/useResizeObserver.spec.tsx

```diff
@@ -26,22 +27,26 @@ jest.mock("resize-observer-polyfill", () => ({
 describe("useResizeObserver", () => {
   it("Calls the callback with the expected entries", async () => {
     const callback = jest.fn().mockImplementation(value => value)
-    const { result, waitForNextUpdate, unmount } = renderHook(() =>
-      useResizeObserver(callback)
-    )
+    const { result, unmount } = renderHook(() => useResizeObserver(callback))
     await act(async () => {
```

Comment:
This seems to be the problem now @HeartSquared
```
Warning: The current testing environment is not configured to support act(...)
```

Unsure about this fix 🤔 
https://github.com/testing-library/react-testing-library/issues/1061#issuecomment-1119435440

Reply:
Deprecated in favour of native DOM API:
https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver